users/tests/test_views.py

```diff
@@ -557,3 +558,36 @@ def test_menu_expansion_update_should_update_menu_expansion_setting_in_user_pref
 
         user_menu_expansion_setting_after_update = CustomUserPreferences.objects.get(user=self.employee).expanded_menu
         self.assertEqual(user_menu_expansion_setting_after_update, not expanded_menu)
+
+
+class ResetPasswordTests(TestCase):
+    def setUp(self) -> None:
+        self.user_password = "Password1234!"
+        self.user = UserFactory()
+        self.user.set_password(self.user_password)
+        self.user.save()
+
```

Comment:
Why do you left here blank space? ;)

Reply:
Hmmm... good question. This is my bad. Sometimes I put blank space unnecessarily.

Reply:
Done